input.cpp

```diff
@@ -752,7 +752,6 @@ namespace marxan {
         tpf1 = 0;
         tpf2 = 0;
         repeats = 0;
-        asymmetricconnectivity = 0;

```

Comment:
whats' the reason for removing this?

Reply:
Reverted as not related to this pull request.